### PR TITLE
fix build

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-5<br><br>- [ ] `package.json:version` has been updated with `npm version patch` (or `major/minor` as appropriate)<br>- [ ] `@brainhubeu/react-carousel` version has been updated in `docs-www/package.json` to the same which is in `package.json:version`<br><br>

### CODE REVIEW

File: docs-www/.babelrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ],
+  "env": {
+    "production": {
+      "plugins": [
+        "transform-react-constant-elements",
+        "transform-react-remove-prop-types"
+      ]
+    }
+  }
+}
```
Comment (RobertHebel) [inline docs-www/.babelrc]: I thought gatsby has babel support out of the box https://www.gatsbyjs.org/docs/babel/. Isn't it true?

Comment (piotr-s-brainhub) [reply docs-www/.babelrc]: For the following `.babelrc`:
```
{
  "presets": ["babel-preset-gatsby"]
}
```
or
```
{
  "plugins": ["@babel/plugin-proposal-optional-chaining"],
  "presets": [
    [
      "babel-preset-gatsby",
      {
        "targets": {
          "browsers": [">0.25%", "not dead"]
        }
      }
    ]
  ]
}
```
or any other containg the `babel-preset-gatsby` preset, I get the following error:
```
⟩ yarn build
yarn run v1.16.0
warning ../../../../package.json: No license field
$ rm -rf .cache && rm -rf public && NODE_ENV=test gatsby build --prefix-paths
success delete html and css files from previous builds — 0.015 s
success open and validate gatsby-config — 0.008 s
success copy gatsby files — 0.019 s
success onPreBootstrap — 0.245 s
success source and transform nodes — 0.162 s
success building schema — 0.411 s
success createLayouts — 0.014 s
success createPages — 0.048 s
success createPagesStatefully — 0.016 s
success onPreExtractQueries — 0.001 s
success update schema — 0.162 s
success extract queries from components — 0.064 s
success run graphql queries — 0.264 s
success write out page data — 0.003 s
success write out redirect data — 0.001 s
success onPostBootstrap — 0.001 s

info bootstrap finished - 2.977 s

success Building CSS — 0.471 s

error Generating JavaScript bundles failed


  Error: ./.cache/production-app.js
  Module build failed: Error: `babel-preset-gatsby` has been loaded, which consumes config generated by the Gatsby CLI. Set `NODE_ENV=test` to bypass, or run `gatsby build` first. (While processing preset  : "/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-preset-gatsby/index.js")
      at loadCachedConfig (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-preset-gatsby/index.js:18:15)
      at preset (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-preset-gatsby/index.js:34:29)
      at /Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/options/option-manager.js:317:46
      at Array.map (<anonymous>)
      at OptionManager.resolvePresets (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/options/option-manager.js:275:20)
      at OptionManager.mergePresets (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/options/option-manager.js:264:10)
      at OptionManager.mergeOptions (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/options/option-manager.js:249:14)
      at OptionManager.init (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
      at File.initOptions (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/index.js:212:65)
      at new File (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/file/index.js:135:24)
      at Pipeline.transform (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
      at transpile (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-loader/lib/index.js:46:20)
      at /Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-loader/lib/fs-cache.js:79:18
      at ReadFileContext.callback (/Users/piotr.sroczkowski/brainhub/open-source/react-carousel/docs-www/node_modules/babel-loader/lib/fs-cache.js:15:14)
      at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:237:13)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Setting `NODE_ENV=test` doesn't fix it.

Comment (piotr-s-brainhub) [reply docs-www/.babelrc]: or maybe it works only for gatsby@2

Comment (RobertHebel) [reply docs-www/.babelrc]: ok, so let's keep it as it is for now. I will try to find a solution on Monday and if I find any I will create new PR

**Pull request merged**